### PR TITLE
Jul 31

### PR DESCRIPTION
**Pull request by nvmbrasserie**

### CODE REVIEW

File: datasets/fr/tresor/fr_tresor_gels_avoir.yml [hunk 1]
```diff
@@ -4195,3 +4195,161 @@ lookups:
...
+          - Corps des gardiens de la révolution islamique (IRGC)
+        props:
+          jurisdiction: ir     
+          incorporationDate:
+            - 2000
+            - 2008
+          websites:
+            - pouyaair.com
+            - yasair.com
+          legalForm: compagnie aérienne de fret
```
Comment (jbothma) [inline datasets/fr/tresor/fr_tresor_gels_avoir.yml:4307]: I'd make this sector

File: datasets/gb/coh/gb_coh_disqualified.yml [hunk 2]
```diff
@@ -69,8 +69,16 @@ lookups:
...
         value: Botswana
       - match: Citizen Of Seychelles
         value: Seychelles
+      - match: 
+          - Brithish
+        value: Great Britain
+  type.date:
+    options:
+      - match: "3018-12-12"
+        value: "2018-12-12"
```
Comment (jbothma) [inline datasets/gb/coh/gb_coh_disqualified.yml:84]: any idea what this was for? is it safe to assume this is what they meant?

File: datasets/at/nbter_sanctions/at_nbter_sanctions.yml [hunk 3]
```diff
@@ -33,7 +36,7 @@ assertions:
   min:
     schema_entities:
       Person: 40
-      Organization: 25
+      Organization: 20
```
Comment (jbothma) [inline datasets/at/nbter_sanctions/at_nbter_sanctions.yml:39]: is this how many entities we'd expect when all the dupes are merged? It's probably sensible to guestimate how many unique entities there are when they're all deduped, and use that for the minimum requirements

**Review by jbothma — COMMENTED** [review]
thanks! more questions than anything

File: datasets/at/nbter_sanctions/at_nbter_sanctions.yml (hunk 3 above)
Comment (nvmbrasserie) [reply datasets/at/nbter_sanctions/at_nbter_sanctions.yml:39]: we have roughly 17 unique entities, I'll set it to this amount then

File: datasets/gb/coh/gb_coh_disqualified.yml (hunk 2 above)
Comment (nvmbrasserie) [reply datasets/gb/coh/gb_coh_disqualified.yml:84]: some entites has ["2018-12-12"] either as a startDate or a listingDate, I'd assume that's the case

File: datasets/fr/tresor/fr_tresor_gels_avoir.yml (hunk 1 above)
Comment (jbothma) [inline datasets/fr/tresor/fr_tresor_gels_avoir.yml:4307]:
```suggestion
          sector: compagnie aérienne de fret
```

**Pull request merged**